packages/truffle/test/scenarios/commands/develop.js

```diff
@@ -68,6 +68,50 @@ describe("truffle develop", function () {
     });
   });
 
+  describe("Improper truffle commands", async function () {
+    this.timeout(7000);
+
+    [
+      {
+        cmd: "console",
+        expectedError: `ℹ️ : 'console' is not allowed within Truffle REPL`
+      },
+      {
+        cmd: "CONSOLE",
```

Comment:
Curious about how you thought to add `CONSOLE` as well.

Reply:
[See this comment](https://github.com/trufflesuite/truffle/pull/5579/files#r1002058952)